tests/integration-tests/tests/schedulers/test_slurm.py

```diff
@@ -1062,7 +1062,7 @@ def _test_protected_mode(scheduler_commands, remote_command_executor, cluster):
         ],
     )
     # Assert bootstrap failure queues are inactive and compute fleet status is PROTECTED
-    assert_that(cluster.status()).contains("ComputeFleetStatus: PROTECTED")
+    assert_that(cluster.describe_cluster()).contains("ComputeFleetStatus: PROTECTED")
```

Comment:
Looks like this and the below line can be changed into a `check_status` call.

Reply:
Done